### PR TITLE
fix(cloud): Small fixes for compose commands

### PR DESCRIPTION
**Pull request by craciunoiuc**
### Prerequisite checklist

  - [ ] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
  - [ ] Tested your changes against relevant architectures and platforms;
  - [ ] Ran `make fmt` on your commit series before opening this PR;
  - [ ] Updated relevant documentation.

### CODE REVIEW

File: internal/cli/kraft/cloud/compose/start/start.go [hunk 1]
```diff
@@ -117,7 +118,12 @@ func (opts *StartOptions) Run(ctx context.Context, args []string) error {
 			return fmt.Errorf("service '%s' not found", serviceName)
 		}
 
-		instances = append(instances, service.Name)
+		name := strings.ReplaceAll(fmt.Sprintf("%s-%s", opts.Project.Name, service.Name), "_", "-")
+		if cname := service.ContainerName; len(cname) > 0 {
```
Comment (razvand) [inline internal/cli/kraft/cloud/compose/start/start.go:122]: Why is this condition for? Is there a `containername` field name in the Compose specification?

**Review by razvand — COMMENTED** [review]
Thanks, @craciunoiuc, this works.

I have a small question below. Then I'll approve.

File: internal/cli/kraft/cloud/compose/start/start.go (hunk 1 above)
Comment (craciunoiuc) [reply internal/cli/kraft/cloud/compose/start/start.go:122]: yup

**Review by razvand — APPROVED** [review]
Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>

**Review by nderjung — APPROVED** [review]

**Pull request merged**